### PR TITLE
Support active and inactive state on validator sub

### LINKED ISSUE #1142: Support Active and Inactive state on ValidatorSub
- Support active and inactive validator on ValidatorSub
- Wire up to All Validator(toggle button) and validator index page

### PR DESCRIPTION
**Pull request by prin-r**
Fix #1142

### CODE REVIEW

File: scan/src/subscriptions/ValidatorSub.re [hunk 1]
```diff
@@ -124,8 +124,8 @@ module SingleConfig = [%graphql
 
 module MultiConfig = [%graphql
   {|
-      subscription Validator($limit: Int!, $offset: Int!) {
-        validators(limit: $limit, offset: $offset) @bsRecord {
```
Comment (sorawit) [inline scan/src/subscriptions/ValidatorSub.re]: While we are here. Let's also add order by tokens desc.

File: scan/src/pages/ValidatorHomePage.re [hunk 2]
```diff
@@ -94,75 +94,33 @@ let renderBody = (rank, validator: ValidatorSub.t, bondedTokenCount) => {
   // let reportRate = (validator.completedRequestCount |> float_of_int) /. allRequestCount *. 100.;
 
   <TBody key={rank |> string_of_int}>
```
Comment (sorawit) [inline scan/src/pages/ValidatorHomePage.re]: Shouldn't key be the validator address? Don't think rank is the right value for key?

**Review by sorawit — DISMISSED** [review]

File: scan/src/subscriptions/ValidatorSub.re [hunk 3]
```diff
@@ -186,12 +186,12 @@ let get = operator_address => {
   };
 };
 
-let getList = (~page=1, ~pageSize=10, ()) => {
+let getList = (~page=1, ~pageSize=10, ~isActive, ()) => {
```
Comment (evilpeach) [inline scan/src/subscriptions/ValidatorSub.re]: Let's remove the default value of page and pageSize.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by tansawit — APPROVED** [review]

**Pull request merged**